File: semestre-1/algoritmos/lista-4/MR-Alg-04-Ex-03.py
def celsiusToFahrenheit(temp: float):
    return temp * 9/5 + 32

print("Celsius\t| Fahrenheit")
print("--------+-----------")
for i in range(0, 101, 10):
    f = celsiusToFahrenheit(i)

    print(f"{i}\t| {f}")

File: semestre-1/algoritmos/MR-Alg-04/.c/MR-Alg-04-Ex-03.c
/*Escreva um programa Python que mostre uma tabela
de conversão de temperaturas em graus Celsius e graus Fahrenheit. A tabela deve incluir em
suas linhas todas as temperaturas entre 0 e 100 graus Celsius que sejam múltiplas de 10
graus Celsius. Inclua os cabeçalhos apropriados e tabulações para suas colunas. Pesquise na
internet sobre a fórmula de conversão de temperaturas Celsius para Fahrenheit.*/

#include <stdio.h>

void main() {
    printf("Celsius\t| Fahrenheit\n");
    printf("--------+-----------\n");
    for (int i = 0; i <= 100; i += 10) {
        float fahrenheit = i * 9/5 + 32;
        printf("%d\t| %.1f\n", i, fahrenheit);
    }
}
